docs/source/getting-started/what-is-jupyterhub.md

```diff
@@ -0,0 +1,465 @@
+  the machine (they can log in by ssh), they will be able to log in to
+  JupyterHub without any other setup.  Thus, JupyterHub fills the role
+  of a ssh server, but providing a web-browser based way to access the
+  machine.
+
+
+But those are fairly limited, and thus there are [plenty of others to
+choose
+from](https://github.com/jupyterhub/jupyterhub/wiki/Authenticators).
+You can connect to almost any other existing service to manage your
+users.  You either use all users from this other service (e.g. your
+company), or whitelist only the allowed users (e.g. your group's
+Github usernames).  Some other popular authenticators include:
+
+- **OAuthenticator** uses the standard OAuth protocol to verify users.
+  For example, you can easily use Github to authenticate your users -
+  people have a "click to login with Github" button.  This is often
+  done with a whitelist to only allow certain users.
+
+- **NativeAuthenticator** actually stores and validates its own
+  usernames and passwords, unlike most other authenticators.  Thus,
+  you can manage all your users within JupyterHub only.
+
+- There are authenticators for LTI (learning management systems),
+  Shibboleth, Kerberos - and so on.
+
+The authenticator is configured with the
+`c.JupyterHub.authenticator_class` configuration option in the
+`jupyterhub_config.py` file.
+
+The authenticator runs internally to the Hub process but communicates
+with outside services.
+
+If you have trouble logging in, this is usually a problem of the
+authenticator.  The authenticator logs are part of the the JupyterHub
+logs, but there may also be relevant information in whatever external
+services you are using.
+
+### Spawner
+
+The **spawner** ([basics](spawners-basics),
+[reference](../reference/spawners)) is the real core of
+JupyterHub: when someone wants a notebook server, the spawner allocates
+resources and starts the server.  The notebook server could run on the
+same machine as JupyterHub, on another machine, on some cloud service,
+or even more.  They can limit resources (CPU, memory) or isolate users
+from each other - if the spawner supports it.  They can also do no
+limiting and allow any user to access any other user's files if they
+are not configured properly.
+
+Some basic spawners included in JupyterHub is:
+
+- **LocalProcessSpawner** is build into JupyterHub and basically tries
+  to switch user to the given username (`su` (&)) and start the
+  notebook server.  It requires that the hub be run as root (because
+  only root has permission to start processes as other user IDs).
+  LocalProcessSpawner is no different than a user logging in with
+  something like `ssh` and running something.  PAMAuthenticator and
+  LocalProcessSpawner is the most basic way of using JupyterHub (and
+  what it does out of the box) and makes the hub not too dissimilar to
+  an advanced ssh server.
+
+There are many more advanced spawners:
```

Comment:
Rather than listing the other spawners here, I think we should just have a single canonical list of spawners and link to that. If we wanna show a few as examples, pick 1 or 2 at most.